### PR TITLE
feat: User avatar component

### PR DESCRIPTION
**Pull request by iamsivin**
# Pull Request Template

Fixes # (issue)

## How Has This Been Tested?

https://user-images.githubusercontent.com/64252451/201078100-f319d6e6-1c36-46fc-a3fa-0256d0f98da2.mp4

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have tested in both Android and iOS platform
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/components/UserAvatar/UserAvatar.js [hunk 1]
```diff
@@ -0,0 +1,234 @@
+import React, { useMemo } from 'react';
+import { useTheme } from '@react-navigation/native';
+import LinearGradient from 'react-native-linear-gradient';
+import { StyleSheet, View } from 'react-native';
+import PropTypes from 'prop-types';
+import FastImage from 'react-native-fast-image';
+import ImageLoader from './ImageLoader';
```
Comment (muhsin-k) [inline src/components/UserAvatar/UserAvatar.js:7]: Let's add `<ImageLoader/>` later.

File: src/components/UserAvatar/UserAvatar.js [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+import Text from 'components/Text/Text';
```
Comment (muhsin-k) [inline src/components/UserAvatar/UserAvatar.js:8]:
```suggestion
import { Text } from 'components';
```

File: src/components/UserAvatar/UserAvatar.js [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+import { getUserInitial } from 'helpers/index';
```
Comment (muhsin-k) [inline src/components/UserAvatar/UserAvatar.js:9]:
```suggestion
import { getUserInitial } from 'helpers';
```

File: src/components/UserAvatar/UserAvatar.js (hunk 1 above)
Comment (iamsivin) [inline src/components/UserAvatar/UserAvatar.js:7]:
```suggestion

```

**Pull request merged**